crates/common/tedge_config/src/tedge_config_cli/tedge_config.rs

```diff
@@ -444,7 +451,19 @@ define_tedge_config! {
                 #[tedge_config(note = "If set to 'auto', this cleans the local session accordingly the detected version of mosquitto.")]
                 #[tedge_config(example = "auto", default(variable = "AutoFlag::Auto"))]
                 local_cleansession: AutoFlag,
-            }
+            },
+
+            #[tedge_config(default(value = false))]
+            #[doku(skip)] // Hide the configuration in `tedge config list --doc`
+            in_mapper: bool,
+
+            // TODO validation
+            /// The topic prefix that will be used for the mapper bridge MQTT topic. For instance,
+            /// if this is set to "c8y", then messages published to `c8y/s/us` will be
+            /// forwarded by to Cumulocity on the `s/us` topic
+            #[tedge_config(example = "c8y", default(value = "c8y"))]
+            #[doku(skip)] // Hide the configuration in `tedge config list --doc`
+            topic_prefix: TopicPrefix,
```

Comment:
Understood. I don't quite know if/why that makes a difference to this, as this is the local prefix. But this will impact the bridge library, which will need to support remote prefixes